### ISSUE TITLE
Не падает в ошибку шаг инициализации ИБ в случае

### ISSUE THREAD

**Issue by ivanmolodec** [firstBitMarksistskaya/jenkins-lib#131]
При запуске базы в пользовательском режиме обновление падает:
```
[2024-09-24T14:25:15.889Z] 24.09.2024 17:24:42 - 81% Нашли форму обновления подождем еще
[2024-09-24T14:25:15.889Z] ERROR: 24.09.2024 17:25:12 Приложение не было обновлено на новую версию по причине:
[2024-09-24T14:25:15.889Z] Некорректный формат файла: C:\Users\Jenkins\AppData\Local\Temp\v8_2177_1ad.kpitmp\ModelData.zip
[2024-09-24T14:25:15.889Z] ERROR: 24.09.2024 17:25:12 Неудачное обновление конфигурации
[2024-09-24T14:25:15.889Z] ИНФОРМАЦИЯ - Выполнение команды/действия в режиме 1С:Предприятие завершено.
```
При этом шаг зелёный. vanessa-runner v2.2.22 - мб изменились какие-то коды возврата?

**Comment by nixel2007** [comment]
А ты можешь это воспроизвести без либы, просто из терминала?

**Comment by johnnyshut** [comment]
Совершенно случайно похоже знаю как воспроизводится. 
ЕРП УХ (3.2.2.12). На обработчике "ОбновлениеИнформационнойБазыУХ.ПервыйЗапуск" (Видимо в момент чтения метаданных)
Просто закрывается 1С. Воспроизводится только при запуске через vanessa-runner. 
Даже в самом полном ТЖ никаких намеков на ошибку. Только dump. 

Причем второй раз если вызвать vrunner, то уже до конца дойдет. 
![image](https://github.com/user-attachments/assets/a958a2ab-46c4-44af-96d1-36f772f411da)

**Comment by ivanmolodec** [comment]
> А ты можешь это воспроизвести без либы, просто из терминала?

Да, из терминала всё воспроизводится. Вопрос то тут не в ванессе сейчас, а в том что ошибка есть, но либа не считает это ошибкой. Я попробую на небольшом проекте воспроизвести подобную ошибку и посмотреть какой там кода возврата ванесса дает.

> Совершенно случайно похоже знаю как воспроизводится. В ЕРП УХ (3.2.2.12). На обработчике "ОбновлениеИнформационнойБазыУХ.ПервыйЗапуск" (Но с этим не связано) Воспроизводится только при запуске через vanessa-runner. Ни в полном ТЖ, ни в ЖР никаких ошибок. Только dump. https://dropmefiles.com/qbwY4

У меня просто ЕРП и сообщение об ошибке выводится. На багборде ничего про это не нашёл, так что ещё придется покопаться по какой причине конфа не обновляется. Но кстати если просто запустить эту файловую базу то там вообще другая ошибка падает.

**Comment by nixel2007** [comment]
Надо просто понять, кто конкретно это сообщение ERROR выдает. может быть форма обновления, но обработка ЗакрытьПредприятие это не отлавливает и завершает предприятие с кодом возврата 0.

**Comment by nixel2007** [comment]
Врубай отладочный лог враннера

**Comment by nixel2007** [comment]
Посмотрю, но уже наверное в понедельник
